src/config.rs

```diff
@@ -2234,6 +2240,30 @@ impl Default for CdcConfig {
     }
 }
 
+impl CdcConfig {
+    fn validate(&mut self) -> Result<(), Box<dyn Error>> {
+        if self.old_value_cache_size == 0 {
+            return Err("cdc.old-value-cache-size can't be 0".into());
+        }
+        if self.min_ts_interval == ReadableDuration::secs(0) {
+            return Err("cdc.min-ts-interval can't be 0s".into());
+        }
+        if self.incremental_scan_threads == 0 {
+            return Err("cdc.incremental-scan-threads can't be 0".into());
+        }
+        if self.incremental_scan_concurrency == 0 {
+            return Err("cdc.incremental-scan-concurrency can't be 0".into());
+        }
+        if self.incremental_scan_concurrency < self.incremental_scan_threads {
+            return Err(
+                "cdc.incremental-scan-concurrency must be larger than cdc.incremental-scan-threads"
+                    .into(),
+            );
+        }
```

Comment:
No need to check ` self.incremental_scan_concurrency == 0` since `self.incremental_scan_concurrency < self.incremental_scan_threads` contains it